### PR TITLE
Unbond timer

### PR DESCRIPTION
**Pull request by siyushin**
Fixed Issus #64: Compute unbound progress with block number replaces of time.

### CODE REVIEW

File: packages/page-account/src/StakingList.tsx [hunk 1]
```diff
@@ -337,7 +360,8 @@ class Overview extends React.PureComponent<Props, State> {
                 <td>{formatBalance(item.amount)}</td>
                 <td>
                   {/* <div className="textGradient">{formatKtonBalance(ringToKton(item.amount, item.month))}</div> */}
-                  <div className='textGradient'>{this.renderUnbondingStatus(this.getUnbondingEndTime(item.unbonding_at))}</div>
+                  {/* <div className='textGradient'>{this.renderUnbondingStatus(this.getUnbondingEndTime(item.unbonding_at))}</div> */}
+                  <div className='textGradient'>{this.renderUnbondingStatusWithBlockNumber(this.props.currentBlock, this.formatIndex(item.unbonding_extrinsic_index) + this.unbondEnd)}</div>
```
Comment (WoeOm) [inline packages/page-account/src/StakingList.tsx:368]: i18n

File: packages/page-account/src/StakingList.tsx [hunk 2]
```diff
@@ -119,15 +120,30 @@ class Overview extends React.PureComponent<Props, State> {
...
+      idx = parseInt(extrinsicIndex.split('-')[0])
+    } catch (error) {
+      console.error(error)
+    }
+
+    return idx
+  }
+
+  computeUnbondProgress(start: number, current: number, end: number): string {
+    return ((current - start) / (end - start) * 100).toFixed(2) + ' %'
```
Comment (WoeOm) [inline packages/page-account/src/StakingList.tsx]: The upper limit of progress is 100%

Comment (siyushin) [reply packages/page-account/src/StakingList.tsx]: Fixed it in commit 830c04d

File: packages/page-account/src/StakingList.tsx (hunk 1 above)
Comment (siyushin) [reply packages/page-account/src/StakingList.tsx:368]: Fixed it in commit 0f4bbe7...

**Review by WoeOm — APPROVED** [review]

**Pull request merged**